### DISCUSSION TITLE
license does not verify

### DISCUSSION THREAD

**Discussion by Elfyn8** [aclap-dev/video-downloadhelper#356] [Help and Q&A]
### Your question

Hello. i am having issues downloading videos. When i try to register my license, it gets stuck in "verifing the license", also, when i try to download something from youtube, it never starts, also I can't choose the folder where i want to download any videos.

It is like the VDH does not want to start working, also, when i tried to install the VDH again, the file reached the 50% of the downloading bar, it finish suddenly, i can't install the auxiliary app or register my existing license.

As said before, it is like the VDH gets stuck before starting.

Anybody has the same issue?

Thanks for your help

### Addon & CoApp version and details

Video DownloadHelper
Versión 8.2.2.8 (stable)
Idioma del Navegador: es-ES
Compilación de la plataforma
Compilado el 2024-03-18
Opciones de compilación: browser=firefox, linuxlic=false, noyt=false, target=mozilla
⊖
Plataforma
Plataforma Win x86-64
Navegador Mozilla Firefox 124.0.1

**Comment by jcfrog** [comment]
hi,
could it be a firewall or antivirus issue?

**Reply by Elfyn8** [reply]
i checked my antivirus/firewall and made an exception. it seems to be working fine again. i did not have this problem before, it happen after an update, so maybe it was related.

thanks

**Reply by jcfrog** [reply]
yes there are some false positives on our latest releases.
